gradle-plugin/build.gradle

```diff
@@ -2,17 +2,23 @@ plugins {
   id 'groovy'
   id 'java-gradle-plugin'
   id 'maven-publish'
-  id 'com.palantir.git-version' version '0.10.1'
```

Comment:
This dependency is no longer needed, it was the gradle equivalent of `sbt-dynver` but as can be seen from the referenced github issue below it didn't even work properly which is explains all of the manual git calls